flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/job/SubtaskExecutionAttemptDetailsInfo.java

```diff
@@ -131,6 +137,7 @@ public SubtaskExecutionAttemptDetailsInfo(
         this.status = Preconditions.checkNotNull(status);
         this.attempt = attempt;
         this.host = Preconditions.checkNotNull(host);
```

Comment:
will host be not null now that we are using Endpoint? If not should it be null or (unassigned)?